### ISSUE TITLE
Does not update links

### ISSUE THREAD

**Issue by oldmanumby** [derwish-pro/obsidian-unique-attachments#12]
The description claims ... "After using "Rename all attachments" the plugin updates the links in all notes that used these files."

It renamed the files but did not update the links accordingly.

FWI ... Fresh install, using the latest versions of everything.

**Comment by daephx** [comment]
@OldManUmby what kind of links are you using?

- Wikilink: `[[my_image.jpg]]`
- Markdown: `[my_image](./my_image.jpg)`

If you use markdown links, then ignore me...
I'm just coming in here as a new user and got burned because I didn't notice this part of the readme.

> The "Rename all attachments" command is only compatible with standard markdown links (![](../attachments/title.png)). Wikilinks are not supported. You can convert all wikilinks to markdown links with [Consistent attachments and links](https://github.com/derwish-pro/obsidian-consistent-attachments-and-links) plugin. ''Rename only active attachments'' command works with all kind of links and generate links based on your preferences.

A few notes is one thing but 100's and it's caused some busy work to cleanup. Git saved my ass but I believe it should be even more upfront in the readme. As I identify as a dumb user and don't always read things. A potentially destructive action like this needs to be 100% clearly stated.

**Comment by oldmanumby** [comment]
Yep. I saw that. I didn't use wikilinks on the images that did not get renamed. I used standard markdown links. Yet nothing happened.

**Comment by dy-sh** [comment]
Hello. Is the problem still not resolved? Could you send a test vault with sample notes showing the problem?

**Comment by mitsuru793** [comment]
I have the same issue. I use markdown link, but not wiki-link.
